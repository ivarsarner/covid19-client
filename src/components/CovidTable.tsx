import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Table from '@material-ui/core/Table';
import TableBody from '@material-ui/core/TableBody';
import TableCell from '@material-ui/core/TableCell';
import TableContainer from '@material-ui/core/TableContainer';
import TableHead from '@material-ui/core/TableHead';
import TableRow from '@material-ui/core/TableRow';
import Paper from '@material-ui/core/Paper';

interface AmericanStateObject {
  state: string;
  hospitalizedCurrently: number | null;
  deaths: number;
}

interface Props {
  americanStates: AmericanStateObject[];
}

/* const useStyles = makeStyles({
  table: {
    maxWidth: 'md',
  },
}); */

const CovidTable: React.FC<Props> = ({ americanStates }) => {
  //const classes = useStyles();

  return (
    <TableContainer component={Paper}>
      <Table aria-label="Table of Covid-19 cases in America">
        <TableHead>
          <TableRow>
            <TableCell>State</TableCell>
            <TableCell align="right">Currently in hospital</TableCell>
            <TableCell align="right">Latest deaths in 3 days</TableCell>
          </TableRow>
        </TableHead>
        <TableBody>
          {americanStates.map(({ state, hospitalizedCurrently, deaths }) => (
            <TableRow key={state}>
              <TableCell>{state}</TableCell>
              <TableCell align="right">{hospitalizedCurrently || 0}</TableCell>
              <TableCell align="right">{deaths}</TableCell>
            </TableRow>
          ))}
        </TableBody>
      </Table>
    </TableContainer>
  );
};

export default CovidTable;
