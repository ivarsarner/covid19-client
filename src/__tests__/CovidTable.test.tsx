import React from 'react';
import { render, cleanup } from '@testing-library/react';
import CovidTable from '../components/CovidTable';
import mockData from './mockData/americanStatesMock.json';

let container: any;
let getByText: any;

beforeEach(() => {
  const wrapper = render(<CovidTable americanStates={mockData} />);
  container = wrapper.container;
  getByText = wrapper.getByText;
});

afterEach(cleanup);

test('renders CovidTable', () => {
  expect(container).toMatchSnapshot();
});

test('renders table head', () => {
  const tableHeadDeath = getByText('New deaths in the last 3 days');
  expect(tableHeadDeath).toBeInTheDocument();
});

test('renders table head', () => {
  const arkansas = getByText('AK');
  expect(arkansas).toBeInTheDocument();
});
