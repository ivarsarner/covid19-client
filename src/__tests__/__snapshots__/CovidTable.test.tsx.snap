// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders CovidTable 1`] = `
<div>
  <div
    class="MuiPaper-root MuiTableContainer-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <table
      aria-label="Table of Covid-19 cases in America"
      class="MuiTable-root"
    >
      <thead
        class="MuiTableHead-root"
      >
        <tr
          class="MuiTableRow-root MuiTableRow-head"
        >
          <th
            class="MuiTableCell-root MuiTableCell-head"
            scope="col"
          >
            State
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
            scope="col"
          >
            Currently in hospital
          </th>
          <th
            class="MuiTableCell-root MuiTableCell-head MuiTableCell-alignRight"
            scope="col"
          >
            New deaths in the last 3 days
          </th>
        </tr>
      </thead>
      <tbody
        class="MuiTableBody-root"
      >
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            AK
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            8
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            AL
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            18
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            AR
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            64
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            AS
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            AZ
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            713
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            19
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            CA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4555
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            160
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            CO
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            790
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            6
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            CT
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1242
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            93
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            DC
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            447
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            19
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            DE
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            285
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            11
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            FL
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            53
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            GA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            28
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            GU
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            HI
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            IA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            413
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            22
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            ID
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            IL
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4293
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            165
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            IN
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1328
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            61
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            KS
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            5
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            KY
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            394
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            10
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            LA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1324
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            59
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            3128
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            277
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MD
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1640
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            84
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            ME
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            36
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MI
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1437
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            158
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MN
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            434
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            44
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MO
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            824
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            33
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MP
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MS
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            645
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            21
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            MT
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NC
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            442
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            20
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            ND
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            29
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NE
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            6
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NH
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            107
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            17
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NJ
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4308
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            303
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NM
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            198
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            19
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NV
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            444
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            13
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            NY
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            7262
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            433
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            OH
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            962
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            35
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            OK
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            177
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            6
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            OR
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            164
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            3
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            PA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2230
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            91
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            PR
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            4
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            RI
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            283
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            23
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            SC
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            15
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            SD
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            77
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            3
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            TN
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            2
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            TX
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1626
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            84
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            UT
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            144
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            6
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            VA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            1555
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            27
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            VI
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            VT
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            21
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            WA
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            335
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            30
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            WI
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            514
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            16
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            WV
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            49
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            3
          </td>
        </tr>
        <tr
          class="MuiTableRow-root"
        >
          <td
            class="MuiTableCell-root MuiTableCell-body"
          >
            WY
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            12
          </td>
          <td
            class="MuiTableCell-root MuiTableCell-body MuiTableCell-alignRight"
          >
            0
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
